3-UsingAPIs/Week1/homework/ex1-johnWho.js

```diff
@@ -7,20 +7,31 @@ Rewrite this function, but replace the callback syntax with the Promise syntax:
   didn't pass in a first name!"
 ------------------------------------------------------------------------------*/
 // TODO see above
-const getAnonName = (firstName, callback) => {
+/*const getAnonName = (firstName, callback) => {
```

Comment:
I personally don't care at all, but it's considered good practice to clean up unnecessary comments like these as soon as you've got something working :)

Reply:
You are right, I will do it from now, 
Thank you a lot @kelleyvanevert